### DISCUSSION TITLE
How to set data mutable

### DISCUSSION THREAD

**Discussion by tartaros1** [NationalSecurityAgency/ghidra#5179] [Q&A]
Hello all,
I have some data that is written by a function, the address is passed as a parameter (the ref window shows that the ref is of param type). The Data is initial zero. The decompiler seems to not recognize that the data has been overwritten by that function and therefore the decompiler display it as zero value. I think that when the data ref type is not set to WRITE the decompiler thinks that the data is constant zero. How can I fix it? Is there a way to mark the data as mutable?

Thanks in advance

**Comment by mumbel** [comment]
Check your memory map for the address of the data for at least read/write.  (`Window`->`Memory Map`)
